Benchmarks/FourParameters.cs

```diff
@@ -0,0 +1,64 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkMockNet.PCLMock;
+using FakeItEasy;
+using Moq;
+using NSubstitute;
+using Rocks;
+using System;
+
+namespace BenchmarkMockNet.Benchmarks
+{
+   public class FourParameters : MockingBenchmark
+   {
+      private readonly string _a = "a";
+      private readonly Guid _b = Guid.NewGuid();
+      private readonly string _c = "c";
+      private readonly Guid _d = Guid.NewGuid();
```

Comment:
Can we keep these inline like the other tests?

Reply:
We can, especially if there's just an `int` parameter. I was trying to avoid the inclusion of making a `Guid`, which would've been minor anyway, but as we're going to have 1 parameter, it's a moot point.